python/GafferImageUI/OpenColorIOTransformUI.py

```diff
@@ -96,7 +96,8 @@ def colorSpacePresetValues( plug, config = None ) :
 
 			"description",
 			"""
-			Context override for OCIO config.
+			> Warning : Deprecated - please use the `OpenColorIOContext`
+			> node instead.
```

Comment:
Is it worth adding the deprecation message to the `_ContextFooter` button toolTip to steer users towards the new approach?

Reply:
What I tried first was to hide the widget entirely unless it had child plugs from a previous use, but it annoyed me that the `Context` still tab remained visible and I didn't want to go down that rabbit hole. But I would prefer to avoid future usage and the tooltip is easily missed, so I did the extra work to hide the tab in https://github.com/GafferHQ/gaffer/pull/5324/commits/775339269c6fe2b922a62312cf54b0f1887db65e and https://github.com/GafferHQ/gaffer/pull/5324/commits/29ebc72e2ba34ed897d0f97debc86bbaf5b50372 and then added the activator in https://github.com/GafferHQ/gaffer/pull/5324/commits/702f1a3bdb5d3e041441b73db8f371b02660d095.